docs/quickstart.md

```diff
@@ -146,6 +146,84 @@ Delete the table to clean up:
 bin/uc table delete --full_name unity.default.my_table
 ```
 
+## Manage models in Unity Catalog using MLflow
+
+Unity catalog supports the management and governance of ML models as securable assets.  Starting with 
+[MLflow 2.16.1](https://mlflow.org/releases/2.16.1), MLflow offers integrated support for using Unity Catalog as the 
+backing resource for the MLflow model registry.  What this means is that with the MLflow client, you will be able to 
+interact directly with your Unity Catalog service for the creation and access of registered models.
+
+## Setup MLflow for usage with Unity Catalog
+
+In your desired development environment, install MLflow 2.16.1 or higher:
+
+```sh
+$ pip install mlflow>=2.16.1
```

Comment:
This should just be pip install mlflow now that 2.16.1 is published.  Will update.